src/_rules/border.js

```diff
@@ -18,7 +18,7 @@ const borderStyles = [
 export const borders = [
   [/^border$/, handlerBorder],
   [/^border()-(\d+)$/, handlerBorder, { autocomplete: "(border)-<directions>" }],
-  [/^border-([xy])-(\d+)$/, handlerBorder],
+  [/^border-([xy])-?(\d+)?$/, handlerBorder],
```

Comment:
Hmm... what will that first question mark do? Match zero or one "-"? 🤔  And the second one is to make the capture group optional...  meaning this will make it possible to have only `border-x` and `border-y`?

Reply:
But it would also accept `border-x12` ? 🤔

Reply:
Refactored [here](https://github.com/warp-ds/drive/pull/120/commits/942d1c19fe64f53e5323715b278911ad053b97ac)